src/wallet/wallet.h

```diff
@@ -1193,6 +1193,17 @@ class WalletBatchScanner : public BatchScanner {
         const int nHeight);
 };
 
+/**
+ * We have interfaces that only accept an address, so we sometimes use a UA as a proxy for the
+ * associated account. This indicates whether the proxy should allow all receivers or only
+ * non-transparent ones.
+ */
+enum AccountProxy {
```

Comment:
```suggestion
enum class UnifiedAccountProxyPolicy {
```
Although reading the description of the inhabitants of this type, maybe it should actually be called `UnifiedAccountSpendingPolicy`

Reply:
Done.